src/main/scala/ai/privado/audit/DataElementDiscovery.scala

```diff
@@ -162,6 +164,39 @@ object DataElementDiscovery {
     collectionMethodInfoMap.toMap
   }
 
+  def getFileScore(absoluteFileName: String, xtocpg: Try[Cpg]): String = {
+    var score = 0.0
+    xtocpg match {
+      case Success(cpg) => {
+        if (Dataflow.getSources(cpg).file.where(_.name(absoluteFileName)).length > 0) {
+          score += 1
+        }
+        val objectDefinationDirectoryPattern = "(?i)pojo|model|dao|mapper|entity".r
+
+        objectDefinationDirectoryPattern.findFirstMatchIn(absoluteFileName) match {
+          case Some(_) => score += 0.25
+          case None    =>
+        }
+
+        val fileName = absoluteFileName.substring(absoluteFileName.lastIndexOf("/") + 1)
+
+        val datasubjectPattern = "(?i)person|user|customer".r
+
+        datasubjectPattern.findFirstMatchIn(fileName) match {
+          case Some(_) => score += 1
+          case None    =>
+        }
+        score.toString
+      }
+      case Failure(exception) => {
+        println("Failed to calculate file score")
+        logger.debug("Failed to calculate file score", exception)
+        println(exception.printStackTrace())
```

Comment:
print the stack trace in debug mode

Reply:
Addressed here: https://github.com/Privado-Inc/privado-core/pull/606/commits/66666b5123cee6099681607bf440a6786a9166cc